sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -4,8 +4,12 @@
 
 ### Features Added
 
+The `Close` method on AMQP Message Sender and Message Receiver now blocks until the client receives a `DETACH` response from the remote node.
+
 ### Breaking Changes
 
+- The `Close` method on AMQP Message Sender and Message Receiver now accepts an `Azure::Core::Context` parameter. This parameter is defaulted so this change should not affect existing code.
```

Comment:
This one as well, you even say it should not affect the existing code, yet you list it in Breaking Changes.

Reply:
The change from non-blocking close to blocking close can theoretically break clients, I'd rather call out changes as breaking than not, especially at this point in development.